docs/planning/backend/app/services/fragrance_service.py

```
@@ -0,0 +1,290 @@
+        db.add(fragrance)
+        db.flush()
+
+        # Add notes with positions
+        FragranceService._add_notes(db, fragrance.id, data.top_notes, NotePosition.TOP)
+        FragranceService._add_notes(db, fragrance.id, data.heart_notes, NotePosition.HEART)
+        FragranceService._add_notes(db, fragrance.id, data.base_notes, NotePosition.BASE)
+
+        return fragrance
+
+    @staticmethod
+    def _add_notes(db: Session, fragrance_id: int, note_names: List[str], position: NotePosition):
```

Ruff (0.14.10)

Missing return type annotation for staticmethod `_add_notes`

Add return type annotation: `None`

(ANN205)